### PR TITLE
test(radiogroup.feature): update cypress test FE-2988

### PR DESCRIPTION
**Pull request by KatarzynaQA**
### Proposed behaviour

Apply new `url` approach in tests for:
- RadioButton Group(`timing`: `2:53`).

We decided to leave only `radioButtonGroup` tests scenario and don't divide them on `weekly`, `monthly`, `yearly` tests scenario. All missing tests are covered in `radioButtonGroup`

### Current behaviour

Currently we are passing all parameters via `knobs`.  For RadioButtonGroup we have three separate feature files for `weekly`, `monthly`, `yearly` tests scenario

### Checklist

- [ ] Screenshots are included in the PR
- [ ] Carbon implementation and Design System documentation are congruent
- [ ] All themes are supported
- [x] Commits follow our style guide
- [ ] Unit tests added or updated
- [x] Cypress automation tests added or updated
- [ ] Storybook added or updated
- [ ] Typescript `d.ts` file added or updated

### CODE REVIEW

File: cypress/features/regression/experimental/radioButtonGroup.feature [hunk 1]
```diff
@@ -28,4 +28,90 @@ Feature: Experimental RadioButtonGroup component
...
+      | mp150ú¿¡üßä                  | weeklyValueOtherLanguage    |
+      | !@#$%^*()_+-=~[];:.,?{}&"'<> | weeklyValueSpecialCharacter |
+
+  @positive
+  Scenario: Disable RadioButton
+    When I open default "Experimental RadioButton" component in noIFrame with "radioButtonGroup" json from "experimental" using "disabled" object name
+    Then "first" RadioButton component is disabled
+
+  @positive
+  Scenario: Disable and enable RadioButton
```
Comment (DlgSHi) [inline cypress/features/regression/experimental/radioButtonGroup.feature:67]:
```suggestion
  Scenario: Enable RadioButton
```

File: cypress/features/regression/experimental/radioButtonGroup.feature [hunk 2]
```diff
@@ -28,4 +28,90 @@ Feature: Experimental RadioButtonGroup component
...
+    When I open default "Experimental RadioButton" component in noIFrame with "radioButtonGroup" json from "experimental" using "disabledFalse" object name
+    Then "first" RadioButton component is enabled
+
+  @positive
+  Scenario: Enable reverse RadioButton
+    When I open default "Experimental RadioButton" component in noIFrame with "radioButtonGroup" json from "experimental" using "reverse" object name
+    Then "first" RadioButton is set to reverse
+
```
Comment (DlgSHi) [inline cypress/features/regression/experimental/radioButtonGroup.feature:75]: empty line, please delete it

File: cypress/features/regression/experimental/radioButtonGroup.feature [hunk 3]
```diff
@@ -28,4 +28,90 @@ Feature: Experimental RadioButtonGroup component
...
+    When I open default "Experimental RadioButton" component in noIFrame with "radioButtonGroup" json from "experimental" using "reverseFalse" object name
+    Then "first" RadioButton is not set to reverse
+
+  @positive
+  Scenario Outline: Change RadioButton size to <size>
+    When I open default "Experimental RadioButton" component in noIFrame with "radioButtonGroup" json from "experimental" using "<nameOfObject>" object name
+    Then "first" RadioButton size on preview is set to "<size>"
+    Examples:
+      | size  | nameOfObject |
+      | small | smallSize    |
```
Comment (DlgSHi) [inline cypress/features/regression/experimental/radioButtonGroup.feature:88]: we are using naming like: ${prop} ${value}. It should be like `sizeSmall` and `sizeLarge`

File: cypress/features/regression/experimental/radioButtonGroup.feature [hunk 4]
```diff
@@ -28,4 +28,90 @@ Feature: Experimental RadioButtonGroup component
...
+      | mp150ú¿¡üßä                  | fieldHelpOtherLanguage    |
+      | !@#$%^*()_+-=~[];:.,?{}&"'<> | fieldHelpSpecialCharacter |
+
+  @positive
+  Scenario: Enable fieldHelpInline
+    When I open default "Experimental RadioButton" component in noIFrame with "radioButtonGroup" json from "experimental" using "fieldHelpInline" object name
+    Then "first" field help is set to fieldHelpInline and has margin-left set to "0px" and has margin-right "6px"
+
+  @positive
+  Scenario: Enable and disable fieldHelpInline
```
Comment (DlgSHi) [inline cypress/features/regression/experimental/radioButtonGroup.feature:106]:
```suggestion
  Scenario: Disable fieldHelpInline
```

File: cypress/locators/radioButton/index.js [hunk 5]
```diff
@@ -1,17 +1,13 @@
...
   RADIO_BUTTON_COMPONENT,
   RADIO_BUTTON_GROUP_COMPONENT,
   RADIO_BUTTON_FIELDSET_GROUP,
 } from './locators';
 
-// component preview locators
-export const radioButtonComponent = () => cy.iFrame(RADIO_BUTTON_COMPONENT);
-export const radioButtonByPosition = position => cy.iFrame(RADIO_BUTTON).eq(position);
-
 // no Iframe locators
```
Comment (DlgSHi) [inline cypress/locators/radioButton/index.js:7]: do we need this comment if we are using one type of locators ?

File: cypress/support/step-definitions/common-steps.js [hunk 6]
```diff
@@ -274,8 +275,8 @@ Then('fieldHelp on preview is set to {word} in NoIFrame', (text) => {
 });
 
 Then('{string} fieldHelp on preview is set to {word}', (position, text) => {
-  cy.wait(1500, { log: DEBUG_FLAG }); // delayed to ensure it to run on CI
-  fieldHelpPreviewByPosition(positionOfElement(position)).should('have.text', text);
+  // cy.wait(1500, { log: DEBUG_FLAG }); // delayed to ensure it to run on CI
```
Comment (DlgSHi) [inline cypress/support/step-definitions/common-steps.js:278]: if we are not using this line of code - please delete it.

File: cypress/support/step-definitions/common-steps.js [hunk 7]
```diff
@@ -274,8 +275,8 @@ Then('fieldHelp on preview is set to {word} in NoIFrame', (text) => {
...
+  fieldHelpPreviewNoIFrame(positionOfElement(position)).should('have.text', text);
```
Comment (DlgSHi) [inline cypress/support/step-definitions/common-steps.js:279]: shouldn't it be like: 
`fieldHelpPreviewNoIFrame` -> `fieldHelpPreviewByPositionNoIFrame` ?

File: cypress/support/step-definitions/radio-button-steps.js [hunk 8]
```diff
@@ -1,79 +1,54 @@
-import { fieldHelpPreview, labelByPosition, labelWidthSliderByName } from '../../locators';
+import { labelWidthSliderByName, labelByPositionNoIFrame } from '../../locators';
```
Comment (DlgSHi) [inline cypress/support/step-definitions/radio-button-steps.js:1]:
```suggestion
import {
labelWidthSliderByName,
labelByPositionNoIFrame
} from '../../locators';
```

File: cypress/support/step-definitions/radio-button-steps.js [hunk 9]
```diff
@@ -1,79 +1,54 @@
...
 
 Then('{string} radioButton on preview is {word}', (position, text) => {
-  labelByPosition(positionOfElement(position)).should('have.text', text);
+  labelByPositionNoIFrame(positionOfElement(position)).should('have.text', text);
 });
 
-Then('{string} RadioButton has value {word}', (radioButtonName, text) => {
-  cy.radioButton(radioButtonName, RADIOBUTTON_INPUT_CSS).should('have.attr', 'value').should('contain', text);
+Then('{string} RadioButton has value {word}', (position, text) => {
+  radioButtonComponentByPositionNoIframe(positionOfElement(position)).find('input').should('have.attr', 'value').should('contain', text);
```
Comment (DlgSHi) [inline cypress/support/step-definitions/radio-button-steps.js:16]: We should move 
radioButtonComponentByPositionNoIframe(index).find('input') this to the `index.js` file and use here only the 
`radioButtonComponentByPositionNoIframe(positionOfElement(position))`

File: cypress/support/step-definitions/radio-button-steps.js [hunk 10]
```diff
@@ -1,79 +1,54 @@
...
 });
 
-Then('{string} RadioButton component is disabled', (radioButtonName) => {
-  cy.radioButton(radioButtonName, RADIOBUTTON_INPUT_CSS).should('have.attr', 'type', 'radio')
-    .and('be.disabled')
-    .and('have.attr', 'disabled');
+Then('{string} RadioButton component is disabled', (position) => {
+  labelByPositionNoIFrame(positionOfElement(position)).should('have.attr', 'disabled');
```
Comment (DlgSHi) [inline cypress/support/step-definitions/radio-button-steps.js:20]: .and('be.disabled')?

File: cypress/support/step-definitions/radio-button-steps.js [hunk 11]
```diff
@@ -1,79 +1,54 @@
...
 });
 
-Then('{string} RadioButton component is enabled', (radioButtonName) => {
-  cy.radioButtonComponent(radioButtonName).should('not.be.disabled').and('not.have.attr', 'disabled');
-  cy.radioButton(radioButtonName, RADIOBUTTON_INPUT_CSS).should('have.attr', 'type', 'radio')
-    .and('not.be.disabled')
-    .and('not.have.attr', 'disabled');
+Then('{string} RadioButton component is enabled', (position) => {
+  labelByPositionNoIFrame(positionOfElement(position)).should('not.have.attr', 'disabled');
```
Comment (DlgSHi) [inline cypress/support/step-definitions/radio-button-steps.js:24]: and the same here - `.and('not.be.disabled')`?

File: cypress/support/step-definitions/radio-button-steps.js [hunk 12]
```diff
@@ -1,79 +1,54 @@
...
 });
 
-Then('{string} RadioButton is set to reverse', (radioButtonName) => {
-  cy.radioButton(radioButtonName, 'div:nth-child(2) input').should('have.attr', 'role', 'radio');
+Then('{string} RadioButton is set to reverse', (position) => {
+  radioButtonComponentByPositionNoIframe(positionOfElement(position)).find('div:nth-child(2) input').should('have.attr', 'role', 'radio');
```
Comment (DlgSHi) [inline cypress/support/step-definitions/radio-button-steps.js:28]: here also move `.find(div:nth-child(2) input)` into the `index.js` file

File: cypress/support/step-definitions/radio-button-steps.js [hunk 13]
```diff
@@ -1,79 +1,54 @@
...
 });
 
-Then('{string} RadioButton is not set to reverse', (radioButtonName) => {
-  cy.radioButton(radioButtonName, RADIOBUTTON_INPUT_CSS).should('have.attr', 'role', 'radio');
+Then('{string} RadioButton is not set to reverse', (position) => {
+  radioButtonComponentByPositionNoIframe(positionOfElement(position)).find('input').should('have.attr', 'role', 'radio');
```
Comment (DlgSHi) [inline cypress/support/step-definitions/radio-button-steps.js:32]: here also move `.find('input')` into the `index.js` file

File: cypress/support/step-definitions/radio-button-steps.js [hunk 14]
```diff
@@ -1,79 +1,54 @@
...
 });
 
-Then('{string} RadioButton size on preview is set to {string}', (radioButtonName, size) => {
+Then('{string} RadioButton size on preview is set to {string}', (position, size) => {
   if (size === 'small') {
-    cy.radioButton(radioButtonName, RADIOBUTTON_INPUT_CSS).should('have.css', 'width', '16px')
+    radioButtonComponentByPositionNoIframe(positionOfElement(position)).find('input').should('have.css', 'width', '16px')
```
Comment (DlgSHi) [inline cypress/support/step-definitions/radio-button-steps.js:37]: here also move `.find('input')` into the `index.js` file

File: cypress/support/step-definitions/radio-button-steps.js [hunk 15]
```diff
@@ -1,79 +1,54 @@
...
       .and('have.css', 'height', '16px');
   } else {
-    cy.radioButton(radioButtonName, RADIOBUTTON_INPUT_CSS).should('have.css', 'width', '24px')
+    radioButtonComponentByPositionNoIframe(positionOfElement(position)).find('input').should('have.css', 'width', '24px')
```
Comment (DlgSHi) [inline cypress/support/step-definitions/radio-button-steps.js:40]: here also move `.find('input')` into the `index.js` file

File: cypress/support/step-definitions/radio-button-steps.js [hunk 16]
```diff
@@ -1,79 +1,54 @@
...
       .and('have.css', 'height', '24px');
   }
 });
 
-Then('{string} field help is set to fieldHelpInline and has margin-left set to {string} and has margin-right {string}', (radioButtonName, marginLeft, marginRight) => {
-  cy.radioButton(radioButtonName, 'div div span[data-element="help"]').should('have.css', 'margin-left', marginLeft)
+Then('{string} field help is set to fieldHelpInline and has margin-left set to {string} and has margin-right {string}', (position, marginLeft, marginRight) => {
+  radioButtonComponentByPositionNoIframe(positionOfElement(position)).find(FIELD_HELP_PREVIEW).should('have.css', 'margin-left', marginLeft)
```
Comment (DlgSHi) [inline cypress/support/step-definitions/radio-button-steps.js:46]: here also move `.find(FIELD_HELP_PREVIEW)` into the `index.js` file

Comment (DlgSHi) [inline cypress/support/step-definitions/radio-button-steps.js:46]: here also move `.find(FIELD_HELP_PREVIEW)` into the `index.js` file

File: cypress/support/step-definitions/radio-button-steps.js [hunk 17]
```diff
@@ -1,79 +1,54 @@
...
-
-Then('{string} RadioButton {string} inputWidth is set to {string}', (position, name, width) => {
-  radioButtonByPosition(positionOfElement(position)).parent()
-    .should('have.css', 'width', `${width}px`);
-});
-
-Then('{string} RadioButton label width is set to {string}', (radioButtonName, width) => {
-  cy.radioButton(radioButtonName, 'label').should('have.css', 'width', `${width}px`);
+Then('{string} field help is not set to fieldHelpInline and has margin-left set to {string}', (position, marginLeft) => {
+  radioButtonComponentByPositionNoIframe(positionOfElement(position)).find(FIELD_HELP_PREVIEW).should('have.css', 'margin-left', marginLeft);
```
Comment (DlgSHi) [inline cypress/support/step-definitions/radio-button-steps.js:51]: here also move `.find(FIELD_HELP_PREVIEW)` into the `index.js` file

File: cypress/support/step-definitions/radio-button-steps.js [hunk 18]
```diff
@@ -1,79 +1,54 @@
...
-const RADIOBUTTON_INPUT_CSS = 'div:nth-child(1) input';
-
-Then('fieldHelpInline is enabled', () => {
-  fieldHelpPreview().should('have.class', INLINE);
-});
-
-Then('fieldHelpInline is disabled', () => {
-  fieldHelpPreview().should('not.have.class', INLINE);
-});
+import { FIELD_HELP_PREVIEW } from '../../locators/locators';
```
Comment (DlgSHi) [inline cypress/support/step-definitions/radio-button-steps.js:9]: we should avoid using imports from `locators` in `steps.` Move it to `index.js` file please.

Comment (KatarzynaQA) [reply cypress/support/step-definitions/radio-button-steps.js:9]: fixed

File: cypress/support/step-definitions/radio-button-steps.js (hunk 17 above)
Comment (KatarzynaQA) [reply cypress/support/step-definitions/radio-button-steps.js:51]: fixed

File: cypress/support/step-definitions/radio-button-steps.js (hunk 16 above)
Comment (KatarzynaQA) [reply cypress/support/step-definitions/radio-button-steps.js:46]: fixed

Comment (KatarzynaQA) [reply cypress/support/step-definitions/radio-button-steps.js:46]: fixed

File: cypress/support/step-definitions/radio-button-steps.js (hunk 15 above)
Comment (KatarzynaQA) [reply cypress/support/step-definitions/radio-button-steps.js:40]: fixed

File: cypress/support/step-definitions/radio-button-steps.js (hunk 14 above)
Comment (KatarzynaQA) [reply cypress/support/step-definitions/radio-button-steps.js:37]: fixed

File: cypress/support/step-definitions/radio-button-steps.js (hunk 13 above)
Comment (KatarzynaQA) [reply cypress/support/step-definitions/radio-button-steps.js:32]: fixed

File: cypress/support/step-definitions/radio-button-steps.js (hunk 12 above)
Comment (KatarzynaQA) [reply cypress/support/step-definitions/radio-button-steps.js:28]: it's not a reusable locator, so I prefer to leave it as is

File: cypress/support/step-definitions/radio-button-steps.js (hunk 11 above)
Comment (KatarzynaQA) [reply cypress/support/step-definitions/radio-button-steps.js:24]: fixed

File: cypress/support/step-definitions/radio-button-steps.js (hunk 10 above)
Comment (KatarzynaQA) [reply cypress/support/step-definitions/radio-button-steps.js:20]: it doesn't work here

File: cypress/support/step-definitions/radio-button-steps.js (hunk 9 above)
Comment (KatarzynaQA) [reply cypress/support/step-definitions/radio-button-steps.js:16]: fixed

File: cypress/support/step-definitions/common-steps.js (hunk 6 above)
Comment (KatarzynaQA) [reply cypress/support/step-definitions/common-steps.js:278]: fixed

File: cypress/locators/radioButton/index.js (hunk 5 above)
Comment (KatarzynaQA) [reply cypress/locators/radioButton/index.js:7]: normally we have `// component preview locators` so I replaced it

File: cypress/features/regression/experimental/radioButtonGroup.feature (hunk 4 above)
Comment (KatarzynaQA) [reply cypress/features/regression/experimental/radioButtonGroup.feature:106]: fixed

File: cypress/features/regression/experimental/radioButtonGroup.feature (hunk 3 above)
Comment (KatarzynaQA) [reply cypress/features/regression/experimental/radioButtonGroup.feature:88]: fixed

File: cypress/features/regression/experimental/radioButtonGroup.feature (hunk 2 above)
Comment (KatarzynaQA) [reply cypress/features/regression/experimental/radioButtonGroup.feature:75]: fixed

File: cypress/features/regression/experimental/radioButtonGroup.feature (hunk 1 above)
Comment (KatarzynaQA) [reply cypress/features/regression/experimental/radioButtonGroup.feature:67]: fixed

File: cypress/locators/radioButton/index.js [hunk 19]
```diff
@@ -1,17 +1,16 @@
...
-
-// no Iframe locators
 export const radioButtonFieldset = () => cy.get(RADIO_BUTTON_FIELDSET_GROUP).children();
 export const radioButtonLegend = () => cy.get(RADIO_BUTTON_FIELDSET_GROUP).find('legend').parent();
 export const radioButtonGroupNoIframe = () => cy.get(RADIO_BUTTON_GROUP_COMPONENT);
 export const radioButtonComponentByPositionNoIframe = position => cy.get(RADIO_BUTTON_COMPONENT)
   .eq(position);
+export const radioButtonComponentNoIframe = () => cy.get(RADIO_BUTTON_COMPONENT);
+export const radioButtonInputByPosition = position => radioButtonComponentByPositionNoIframe(position).find('input');
+export const radioButtonFieldHelp = position => radioButtonComponentByPositionNoIframe(position).find(FIELD_HELP_PREVIEW);
```
Comment (DlgSHi) [inline cypress/locators/radioButton/index.js:16]:
```suggestion
export const radioButtonFieldHelpByPositionNoIFrame = position => radioButtonComponentByPositionNoIframe(position).find(FIELD_HELP_PREVIEW);
```

File: cypress/locators/radioButton/index.js [hunk 20]
```diff
@@ -1,17 +1,16 @@
...
-export const radioButtonByPosition = position => cy.iFrame(RADIO_BUTTON).eq(position);
-
-// no Iframe locators
 export const radioButtonFieldset = () => cy.get(RADIO_BUTTON_FIELDSET_GROUP).children();
 export const radioButtonLegend = () => cy.get(RADIO_BUTTON_FIELDSET_GROUP).find('legend').parent();
 export const radioButtonGroupNoIframe = () => cy.get(RADIO_BUTTON_GROUP_COMPONENT);
 export const radioButtonComponentByPositionNoIframe = position => cy.get(RADIO_BUTTON_COMPONENT)
   .eq(position);
+export const radioButtonComponentNoIframe = () => cy.get(RADIO_BUTTON_COMPONENT);
+export const radioButtonInputByPosition = position => radioButtonComponentByPositionNoIframe(position).find('input');
```
Comment (DlgSHi) [inline cypress/locators/radioButton/index.js:15]:
```suggestion
export const radioButtonInputByPositionNoIFrame = position => radioButtonComponentByPositionNoIframe(position).find('input');
```

File: cypress/support/step-definitions/common-steps.js (hunk 7 above)
Comment (DlgSHi) [reply cypress/support/step-definitions/common-steps.js:278]: isn't fixed

File: cypress/support/step-definitions/radio-button-steps.js [hunk 21]
```diff
@@ -1,79 +1,56 @@
...
-Then('{string} RadioButton is set to reverse', (radioButtonName) => {
-  cy.radioButton(radioButtonName, 'div:nth-child(2) input').should('have.attr', 'role', 'radio');
+Then('{string} RadioButton is set to reverse', (position) => {
+  radioButtonComponentByPositionNoIframe(positionOfElement(position)).find('div:nth-child(2) input').should('have.attr', 'role', 'radio');
 });
 
-Then('{string} RadioButton is not set to reverse', (radioButtonName) => {
-  cy.radioButton(radioButtonName, RADIOBUTTON_INPUT_CSS).should('have.attr', 'role', 'radio');
+Then('{string} RadioButton is not set to reverse', (position) => {
+  radioButtonInputByPosition(positionOfElement(position)).should('have.attr', 'role', 'radio');
```
Comment (DlgSHi) [inline cypress/support/step-definitions/radio-button-steps.js:34]:
```suggestion
  radioButtonInputByPositionNoIFrame(positionOfElement(position)).should('have.attr', 'role', 'radio');
```

File: cypress/support/step-definitions/radio-button-steps.js [hunk 22]
```diff
@@ -1,79 +1,56 @@
...
 });
 
-Then('{string} RadioButton size on preview is set to {string}', (radioButtonName, size) => {
+Then('{string} RadioButton size on preview is set to {string}', (position, size) => {
   if (size === 'small') {
-    cy.radioButton(radioButtonName, RADIOBUTTON_INPUT_CSS).should('have.css', 'width', '16px')
+    radioButtonInputByPosition(positionOfElement(position)).should('have.css', 'width', '16px')
```
Comment (DlgSHi) [inline cypress/support/step-definitions/radio-button-steps.js:39]:
```suggestion
    radioButtonInputByPositionNoIFrame(positionOfElement(position)).should('have.css', 'width', '16px')
```

File: cypress/support/step-definitions/radio-button-steps.js [hunk 23]
```diff
@@ -1,79 +1,56 @@
...
       .and('have.css', 'height', '16px');
   } else {
-    cy.radioButton(radioButtonName, RADIOBUTTON_INPUT_CSS).should('have.css', 'width', '24px')
+    radioButtonInputByPosition(positionOfElement(position)).should('have.css', 'width', '24px')
```
Comment (DlgSHi) [inline cypress/support/step-definitions/radio-button-steps.js:42]:
```suggestion
    radioButtonInputByPositionNoIFrame(positionOfElement(position)).should('have.css', 'width', '24px')
```

File: cypress/support/step-definitions/radio-button-steps.js [hunk 24]
```diff
@@ -1,79 +1,56 @@
...
       .and('have.css', 'height', '24px');
   }
 });
 
-Then('{string} field help is set to fieldHelpInline and has margin-left set to {string} and has margin-right {string}', (radioButtonName, marginLeft, marginRight) => {
-  cy.radioButton(radioButtonName, 'div div span[data-element="help"]').should('have.css', 'margin-left', marginLeft)
+Then('{string} field help is set to fieldHelpInline and has margin-left set to {string} and has margin-right {string}', (position, marginLeft, marginRight) => {
+  radioButtonFieldHelp(positionOfElement(position)).should('have.css', 'margin-left', marginLeft)
```
Comment (DlgSHi) [inline cypress/support/step-definitions/radio-button-steps.js:48]:
```suggestion
  radioButtonFieldHelpNoIFrame(positionOfElement(position)).should('have.css', 'margin-left', marginLeft)
```

File: cypress/support/step-definitions/radio-button-steps.js [hunk 25]
```diff
@@ -1,79 +1,56 @@
...
-
-Then('{string} RadioButton {string} inputWidth is set to {string}', (position, name, width) => {
-  radioButtonByPosition(positionOfElement(position)).parent()
-    .should('have.css', 'width', `${width}px`);
-});
-
-Then('{string} RadioButton label width is set to {string}', (radioButtonName, width) => {
-  cy.radioButton(radioButtonName, 'label').should('have.css', 'width', `${width}px`);
+Then('{string} field help is not set to fieldHelpInline and has margin-left set to {string}', (position, marginLeft) => {
+  radioButtonFieldHelp(positionOfElement(position)).should('have.css', 'margin-left', marginLeft);
```
Comment (DlgSHi) [inline cypress/support/step-definitions/radio-button-steps.js:53]:
```suggestion
  radioButtonFieldHelpNoIFrame(positionOfElement(position)).should('have.css', 'margin-left', marginLeft);
```

File: cypress/locators/radioButton/index.js (hunk 19 above)
Comment (KatarzynaQA) [reply cypress/locators/radioButton/index.js:16]: as we discussed if we using it in `noIframe` the name is e.g `radioButtonFieldHelp`.`

File: cypress/support/step-definitions/radio-button-steps.js (hunk 24 above)
Comment (KatarzynaQA) [reply cypress/support/step-definitions/radio-button-steps.js:48]: as we discussed if we using it in `noIframe` the name is e.g `radioButtonFieldHelp`.`

**Pull request merged**